test/cpp/interop/observability_client.cc

```diff
@@ -214,6 +220,35 @@ int main(int argc, char** argv) {
     }
   }
 
+  // TODO(stanleycheung): switch to CsmObservabilityBuilder once xds setup is
+  // ready
+  if (absl::GetFlag(FLAGS_enable_otel_plugin)) {
+    gpr_log(GPR_DEBUG, "Registering Prometheus exporter");
+    opentelemetry::exporter::metrics::PrometheusExporterOptions opts;
+    // default was "localhost:9464" which causes connection issue across GKE
+    // pods
+    opts.url = "0.0.0.0:9464";
+    auto prometheus_exporter =
+        opentelemetry::exporter::metrics::PrometheusExporterFactory::Create(
+            opts);
+    auto meter_provider =
+        std::make_shared<opentelemetry::sdk::metrics::MeterProvider>();
+    meter_provider->AddMetricReader(std::move(prometheus_exporter));
+    grpc::internal::OpenTelemetryPluginBuilder otel_builder;
+    otel_builder.EnableMetric(
```

Comment:
you don't need to set these

Reply:
they'll be enabled by default

Reply:
Just setting the meter provider and registering would be enough

Reply:
Removed those lines. Thanks.